### PR TITLE
Added config class

### PR DESCRIPTION
**Pull request by AjayShinde1999**
Changes done for created config class

### CODE REVIEW

File: src/main/java/com/profileexample/controller/config/ProfileConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.profileexample.controller.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+import javax.annotation.PostConstruct;
+
+@Profile("dev")
+@Configuration
```
Comment (Manojkumarks02) [inline src/main/java/com/profileexample/controller/config/ProfileConfiguration.java:9]: @AjayShinde1999 why this changes

Comment (AjayShinde1999) [reply src/main/java/com/profileexample/controller/config/ProfileConfiguration.java:9]: @Manojkumarks02 Changes done for config

**Review by Manojkumarks02 — CHANGES_REQUESTED** [review]
CHNGE THE impementation
